docker/build_image

```diff
@@ -81,7 +81,7 @@ function addToDigest() {
 }
 function recursiveDigest() {
   if [[ -n "$digestOnly" ]]; then
-     find "$1" -type f ! -path '*/node_modules/*' -exec md5sum {} \;
+     find "$1" -type f ! -path '*/node_modules/*' ! -path '*/.npmrc' ! -path '*/.bundle/config' -exec md5sum {} \;
```

Comment:
This also reduces unnecessary etna builds.  When the docker stuff is deployed all of this goes away!  The etna-gem will just be a local package reference!